clusterloader2/testing/load/modules/measurements.yaml

```diff
@@ -158,6 +159,21 @@ steps:
       - name: Perc50
         query: quantile_over_time(0.50, sum by (container) (container_memory_working_set_bytes / 1024 / 1024)[%v:])
 {{end}}
+{{if $ENABLE_TERMINATED_WATCHES_MEASUREMENT}}
+  - Identifier: TerminatedWatchesMetrics
+    Method: GenericPrometheusQuery
+    Params:
+      action: {{$action}}
+      metricName: Terminated Watches
+      metricVersion: v1
+      dimensions:
+      - resource
+      queries:
+      - name: Watch connections terminated by the kube-apiserver
+        query: sum(increase(apiserver_terminated_watchers_total[%v:])) by (resource)
+      - name: Watch connections reinitialized by the kube-apiserver
+        query: sum(increase(apiserver_watch_cache_initializations_total[%v:])) by (resource)
```

Comment:
Could we split the second query into a separate measurement (but under the same `{{if $ENABLE_TERMINATED_WATCHES_MEASUREMENT}}` branch) with different `Identifier` and `metricName`, say `WatchCacheInitializations` and `Watch Cache Initializations`, respectively?

Sorry for the late notice about this, I realized it just won't look nice in https://perf-dash.k8s.io/ after we enable it by default in our tests.